test/functional/api/v2/discovery_rules_controller_test.rb

```diff
@@ -38,10 +38,11 @@ class Api::V2::DiscoveryRulesControllerTest < ActionController::TestCase
         :name => "foo",
         :search => "bar",
         :hostgroup_id => hostgroup.id,
-        :organization_ids => [organization_one],
+        :organization_ids => [organization_one.id],
```

Comment:
Looks like ID parser in Rails 5 did not catch this anymore, it was a typo anyway. Integer is expected here.

All the rest was just cleaning up Ruby compiler warnings.